data/json/items/ammo/exodii.json

```diff
@@ -3,8 +3,16 @@
     "id": "123ln",
     "type": "AMMO",
     "copy-from": "3006",
-    "name": { "str": "12.3ln round" },
+    "name": { "str": "Exodii rifle round" },
```

Comment:
What does this accomplish? I think most people are going to see A) 12.3ln and recognize that's not an earth cartridge and B) Find it only around Exodii stuff